src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAfterCheck.java

```diff
@@ -139,6 +141,7 @@ public int[] getAcceptableTokens() {
             TokenTypes.LITERAL_TRY,
             TokenTypes.LITERAL_CASE,
             TokenTypes.LAMBDA,
+            TokenTypes.LITERAL_WHEN,
```

Comment:
@nrmancuso I see we made this token default in this check, but not in the other check.